Tuist/ProjectDescriptionHelpers/Module.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Modules.swift
+//  ProjectDescriptionHelpers
+//
+//  Created by Phong on 16/10/2023.
+//
+
+import ProjectDescription
+
+public enum Module: CaseIterable {
+
+    case domain
+    case data
+
+    public var name: String {
+        switch self {
+        case .domain:
+            return "Domain"
+        case .data:
+            return "Data"
+        }
+    }
+
+    public var dependencies: [TargetDependency] {
+        switch self {
+        case .domain:
+            return []
+        case .data:
+            return [.target(name: Module.domain.name)]
+        }
+    }
+
+    public var frameworkPath: String {
+        "\(Constant.modulesRootPath)/\(name)"
+    }
+
+    public var sources: ProjectDescription.SourceFilesList {
+        return ["\(frameworkPath)/\(Constant.sourcesPath)/**"]
+    }
+
+    public var resources: ProjectDescription.ResourceFileElements {
+        switch self {
+        case .data, .domain:
+            return []
+        }
```

Comment:
For shorter:

```suggestion
        []
```

Reply:
solved in [a732c17](https://github.com/nimblehq/ios-templates/pull/536/commits/a732c17059fec3d44aac47494458332c22c60582)